lib/os.js

```diff
@@ -285,6 +286,7 @@ module.exports = {
   type: getOSType,
   userInfo,
   uptime: getUptime,
+  version: getOSVersion,
```

Comment:
Can you look at how e.g. `getOSType()` is imported and exported and follow that pattern? Notably, this is missing the `getCheckedFunction` wrapper and the `SymbolToPrimitive` method.